guides/common/modules/proc_configuring-smart-proxy-for-host-registration-and-provisioning.adoc

```diff
@@ -0,0 +1,58 @@
+[id="configuring-{smart-proxy-context}-for-host-registration-and-provisioning_{context}"]
+= Configuring {SmartProxy} for Host Registration and Provisioning
+
+Use this procedure to configure {SmartProxy} so that you can register and provision hosts using the {SmartProxyServer} instead of the {ProjectServer}.
```

Comment:
```suggestion
Use this procedure to configure {SmartProxy} so that you can register and provision hosts using your {SmartProxyServer} instead of your {ProjectServer}.
```